crates/typst-library/src/text/mod.rs

```diff
@@ -707,6 +727,31 @@ cast! {
     },
 }
 
+/// An ISO 15924-type script identifier
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+pub struct WritingScript(Tag);
+
+impl WritingScript {
+    /// Create from a string representation of an ISO 15924-type code.
+    pub fn new(tag: EcoString) -> Self {
```

Comment:
This can just be `&str` or maybe rather just keep it exactly parallel to `Lang` and `Region`

Reply:
Made this parallel to `Lang`, `Region`.